src/PasswordChecker.php

```diff
@@ -50,19 +63,14 @@ private function isPasswordObvious(string $password): bool
         return is_numeric(str_replace([' ', '/', '-'], '', $password)); // Could be a DOB or phone number
     }
 
-    private function isRecentPassword(string $password, array $recentHashes = []): bool
+    private function isRecentPassword(string $password): bool
     {
-        foreach ($recentHashes as $hash) {
+        foreach ($this->recentHashes as $hash) {
```

Comment:
Will this fail if `setPreviousPasswords` isn't used?

Reply:
@maxakropolis good point ta. Verified, added a test and fix...